src/main/java/picard/sam/SamToFastq.java

```diff
@@ -93,12 +78,28 @@
             "     I=input.bam<br />" +
             "     FASTQ=output.fastq" +
             "</pre>" +
+            "<h4>Tag Usage example:</h4>" +
+            "<br />" +
+            "This will create the fastq from read sequences alongside two fastq files.  One will be converted from the " +
+            " \"CR\" tag with quality from the \"CY\" tag.  The other fastq will be converted from the \"CB\" and \"UR\" " +
+            "tags concatenated together with no separator (not specified on command line) with the qualities coming from " +
+            "the \"CY\" and \"UY\" tags concatenated together." +
+            "<br />" +
+            "<pre>" +
+            "java -jar picard.jar SamToFastq <br />" +
+            "     I=input.bam<br />" +
+            "     FASTQ=output.fastq<br />" +
+            "     SEQUENCE_TAG_GROUP=CR<br />" +
+            "     QUALITY_TAG_GROUP=CY<br />" +
+            "     SEQUENCE_TAG_GROUP=\"CB,UR\"<br />" +
+            "     QUALITY_TAG_GROUP=\"CY,UY\"" +
+            "</pre>" +
             "<hr />";
     @Argument(doc = "Input SAM/BAM file to extract reads from", shortName = StandardOptionDefinitions.INPUT_SHORT_NAME)
     public File INPUT;
 
     @Argument(shortName = "F", doc = "Output FASTQ file (single-end fastq or, if paired, first end of the pair FASTQ).",
-            mutex = {"OUTPUT_PER_RG", "COMPRESS_OUTPUTS_PER_RG"})
+            mutex = {"OUTPUT_PER_RG", "COMPRESS_OUTPUTS_PER_RG", "OUTPUT_DIR"})

```

Comment:
Not sure exactly how `mutex` works but it seems like some of these are redundant, e.g. do you need a mutex on both sides of a relationship? (Is there a place where barclay docs are hosted?)

Reply:
I actually like the redundancy because when you look at the descriptions of each argument you can see what each one is mutually exclusive to